driverstation.py

```diff
@@ -37,6 +37,11 @@ def joystick_stats(joystick):
 Probably we need an object that contains all of the state
 for the joystick?
 """
+# A, B, X, Y, L bumpter, R bumber, back, start, big shinny button
+buttons = [0,0,0,0,0,0,0,0]
+
+# LHand Y, LHand X, L trigger, RHand X, RHand Y, R trigger
+axis_values = [0,0,0,0,0,0]
```

Comment:
You can be robust and ask the joystick how many buttons it has and create the array of that size.   I think a good idea.     Same for axis.     You could store those values in the local API for error checking or something.